sdk/PrebidServerAdapter/PBServerAdapter.m

```diff
@@ -103,121 +114,110 @@ - (NSURLRequest *)buildRequestForAdUnits:(NSArray<PBAdUnit *> *)adUnits {
     }
 }
 
-- (NSDictionary *)requestBodyForAdUnits:(NSArray<PBAdUnit *> *)adUnits {
+- (NSDictionary *)openRTBRequestBodyForAdUnits:(NSArray<PBAdUnit *> *)adUnits {
     NSMutableDictionary *requestDict = [[NSMutableDictionary alloc] init];
 
-    if (self.primaryAdServer == PBPrimaryAdServerMoPub || self.primaryAdServer == PBPrimaryAdServerUnknown) {
-        requestDict[@"cache_markup"] = @(1);
-    }
+    requestDict[@"id"] = @"some-request-id";
+    requestDict[@"test"] = @(1);
+    requestDict[@"tmax"] = @(500);
```

Comment:
can remove this, prebid server has a default 5 seconds if this is not present.

Reply:
fixed this